app/src/main/java/io/foundy/hanstargram/view/posting/PostingViewModel.kt

```diff
@@ -19,6 +19,10 @@ class PostingViewModel : ViewModel() {
         _uiState.update { it.copy(selectedImage = uri) }
     }
 
+    fun editMode() {
```

Comment:
함수이름은 아래처럼 무엇을 하는지 명시하는게 좋아보이네요
```suggestion
    fun changeToEditMode() {
```